tests/resources/dynamic-vars.py

```diff
@@ -31,7 +31,7 @@ def getEnvironment():
         return TestEnvironment.LOCAL
 
 def getName(image):
-    return {TestEnvironment.DRONE: 'harbor.ci.drone.local/library/{}'.format(image),
+        return {TestEnvironment.DRONE: 'wdc-harbor-ci.eng.vmware.com/default-project/{}'.format(image),
             TestEnvironment.LONGEVITY: 'vic-executor1.vcna.io/library/{}'.format(image),
```

Comment:
That would also mean we don't have in-code conditional switches like this as the registry for longevity would be set via environment when kicking off the run.